install_config/registry/securing_and_exposing_registry.adoc

```diff
@@ -209,40 +220,23 @@ Digest: sha256:3662dd821983bc4326bee12caec61367e7fb6f6a3ee547cbaff98f77403cab55
 ----
 
 [[exposing-the-registry]]
-== Exposing the Registry
+== Manually Exposing the Registry
```

Comment:
Given that there is an "Exposing a Non-Secure Registry" section below, shouldn't this be more specifically "Manually Exposing a Secure Registry"?